public/index.html

```diff
@@ -0,0 +1,104 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="UTF-8">
+<title>reCollector: Record Collection Cataloguer</title>
+<link href='https://fonts.googleapis.com/css?family=Righteous' rel='stylesheet' type='text/css'>
+  <link href="../styles/style1.css" type="text/css" rel="stylesheet" media="screen and (min-width:1px) and (max-width:799px)" />
```

Comment:
if you want your server to serve up static files like this one, they'll need to go into the public folder as this is where your static middleware is looking for them